rts/lra/lra-test/lra-test-tck/pom.xml

```diff
@@ -126,6 +110,5 @@
                 <lra.tck.suite.thorntail.trace.params>-Dthorntail.logging=TRACE</lra.tck.suite.thorntail.trace.params>
             </properties>
         </profile>
-       
```

Comment:
formatting

Reply:
I missed this one and adding it back in again will trigger a full rebuild.
But I would also say that since this is pre release code removing the empty line does not obfuscate the intent of the PR and does not impact anyone wanting to diff the code.